quarkus/tests/junit5/src/main/java/org/keycloak/it/utils/DockerKeycloakDistribution.java

```diff
@@ -56,6 +56,8 @@ public void accept(OutputFrame t) {
 
     private static final Logger LOGGER = Logger.getLogger(DockerKeycloakDistribution.class);
 
+    public static final int STARTUP_TIMEOUT = 120;
```

Comment:
nitpicking, add the units to the constant name
```suggestion
    public static final int STARTUP_TIMEOUT_SECONDS = 120;
```